cmd/build-lambda-zip/main_test.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved
+
+package main
+
+import (
+	"archive/zip"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSizes(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+		return
+	}
+	t.Log("Test how different arguments affect binary and archive sizes")
```

Comment:
nit: lowercase `Test` for log format consistency

Reply:
done